src/reconstruction/refinement/addReactionGEM.m

```diff
@@ -69,6 +69,8 @@
 end
 
 nMet = length(model.mets)+1;
+orignMets = numel(model.mets);

```

Comment:
Isn't `length(model.mets)` always equal to `numel(model.mets)`? Couldn't we define `orignMets = nMet + 1`?

I am just curious why once `length` and once `numel` is used for the same purpose.